ganga/GangaCore/Utility/Setup.py

```diff
@@ -1,141 +1,5 @@
-        The value may be a string or a list of strings - in the later case the individual path components will
-        be separated by colons (':').
-        If the path is not defined (see getPackagePath()) then return an empty string.
-        """
-        import os
-        path, tarball = self.getPackagePath(name, force)
-        if path and var in self.packages[name]:
-            from GangaCore.Utility.util import isStringLike
-
-            ppath = self.packages[name][var]
-
-            if isStringLike(ppath):
-                ppath = [ppath]
-
-            return ':'.join([os.path.join(path, p) for p in ppath])
-
-# for p in ppath:
-##                 result += os.path.join(path,p)
-# return result
-        else:
-            return ''
-
-    def prependPath(self, name, var):
-        """ Update environment (os.environ) and prepend a package path to var.
-        """
-        import os.path
-        import sys
-
-        if getExternalHome():
-            path = self.getPackagePath2(name, var)
-            if path:
-                try:
-                    pp = ':' + os.environ[var]
-                except KeyError:
-                    pp = ''
-                os.environ[var] = path + pp
-
-    def setSysPath(self, name, set_python_path=True):
-        """ Update the sys.path variable for a given package by adding a given package to the PYTHONPATH.
-        It assumes that the first item in PYTHONPATH is Ganga, so the package path is added as a second item.
-        If set_python_path is True, it will update the PYTHONPATH as well
-        """
-        import os
-        import sys
-
-        if getExternalHome():
-
-            # get the path(s)
-            paths = self.getPackagePath2(name, 'syspath')
-            if not paths:
-                return True
-
-            # loop over the sys paths to be added
-            for path in reversed(paths.split(':')):
-                if path not in sys.path:
-
-                    # add to sys path
-                    sys.path.insert(1, path)
-
-                # add to PYTHONPATH?
-                if set_python_path:
-                    if 'PYTHONPATH' in os.environ:
-                        pp_toks = os.environ['PYTHONPATH'].split(':')
-                    else:
-                        # PYTHONPATH is empty so add Ganga in first
-                        pp_toks = [sys.path[0]]
-
-                    # create a new PYTHONPATH
-                    if path not in pp_toks:
-                        pp_toks.insert(1, path)
-
-                    os.environ['PYTHONPATH'] = ':'.join(pp_toks)
-
-        return True
-
-    def setPath(self, name, var):
-        """ 
-        Update environment (os.environ) and *set* (overwrite) a package path to var.
-        """
-        import os
-        import sys
-
-        if getExternalHome():
-            path = self.getPackagePath2(name, var)
-            if path:
-                os.environ[var] = path
-
-        return True
-
+class PackageSet
```

Comment:
should this be:

```python
class PackageSetup:
```

Reply:
I missed what @egede said above, looks like this can just go

Reply:
Okayy